libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationViewModel.java

```diff
@@ -207,11 +183,15 @@ void initializeEventDispatcher(NavigationViewEventDispatcher navigationViewEvent
    */
   @SuppressLint("MissingPermission")
   void initialize(NavigationViewOptions options) {
-    NavigationOptions navigationOptions = options.navigationOptions();
-    navigationOptions = navigationOptions.toBuilder().build();
     initializeLanguage(options);
-    initializeTimeFormat(navigationOptions);
     initializeDistanceFormatter(options);
+    NavigationOptions navigationOptions = options.navigationOptions();
+    navigationOptions = navigationOptions.toBuilder()
+        .onboardRouterConfig(buildMapboxOnboardRouterConfig())
```

Comment:
Do we need these? I believe we're not using it here and it'll be available in the `NavigationViewOptions` / `NavigationOptions`. If so we can remove `buildMapboxOnboardRouterConfig`.

Reply:
I've tested with the simulator off using a mock location app to test off route for example. 

I wasn't sure if those options were needed or not. I'll remove them.